packages/waku/src/lib/handlers/dev-worker-api.ts

```diff
@@ -126,67 +125,30 @@ export async function renderRscWithWorker<Context>(
   const id = nextId++;
   const pipe = async () => {
     if (rr.stream) {
-      const reader = rr.stream.getReader();
-      try {
-        let result: ReadableStreamReadResult<unknown>;
-        do {
-          result = await reader.read();
-          if (result.value) {
-            const buf = result.value;
-            let mesg: MessageReq;
-            if (buf instanceof ArrayBuffer) {
-              mesg = { id, type: 'buf', buf, offset: 0, len: buf.byteLength };
-            } else if (buf instanceof Uint8Array) {
-              mesg = {
-                id,
-                type: 'buf',
-                buf: buf.buffer,
-                offset: buf.byteOffset,
-                len: buf.byteLength,
-              };
-            } else {
-              throw new Error('Unexepected buffer type');
-            }
-            worker.postMessage(mesg, [mesg.buf]);
-          }
-        } while (!result.done);
-      } catch (err) {
-        const mesg: MessageReq = { id, type: 'err', err };
-        worker.postMessage(mesg);
-      }
+      worker.postMessage(
+        { id, type: 'pipe', stream: rr.stream } as MessageReq,
+        [rr.stream as unknown as TransferListItem],
+      );
```

Comment:
Can you follow this style and add type assertion only for the necessary part?
```suggestion
      const mesg: MessageReq = { id, type: 'pipe', stream: rr.stream };
      worker.postMessage(
        mesg,
        [rr.stream as unknown as TransferListItem],
      );
```